### PR TITLE
Rename DbConfigFile and Schema

### PR DESCRIPTION
**Pull request by norascheuch**
## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/src/databases/config/db-config-store.ts [hunk 1]
```diff
@@ -32,6 +32,8 @@ import {
 } from "../db-item";
 
 export class DbConfigStore extends DisposableObject {
+  static readonly databaseConfigFileName = "workspace-databases.json";
```
Comment (charisk) [inline extensions/ql-vscode/src/databases/config/db-config-store.ts]: (Optional) I personally like to be explicit with the visibility of properties

```public static readonly databaseConfigFileName = "workspace-databases.json";`

File: extensions/ql-vscode/package.json [hunk 2]
```diff
@@ -92,7 +92,7 @@
     },
     "jsonValidation": [
       {
-        "fileMatch": "workspace-databases.json",
+        "fileMatch": "databases.json",
```
Comment (charisk) [inline extensions/ql-vscode/package.json:95]: I think this is too generic now. It means that if someone has our extension installed, and they open any databases.json file, they'd get JSON validation applied with our schema.

We should look at ways to make it more specific (e.g. by looking at the file path a bit more).

**Review by charisk — COMMENTED** [review]
Thanks for tidying this up! 

Generally LGTM, just have a comment about the JSON validation setup that will need addressing before merging. 

Also can you please add a PR description?

File: extensions/ql-vscode/src/databases/config/db-config-store.ts (hunk 1 above)
Comment (norascheuch) [reply extensions/ql-vscode/src/databases/config/db-config-store.ts]: 👍

**Review by charisk — APPROVED** [review]
🚀

**Pull request merged**